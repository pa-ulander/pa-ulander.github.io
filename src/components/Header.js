import React from 'react'

const Header = () => {
  return (
    <div className="sticky-nav">
      <a href="/" className="sticky-nav-button">
        Home
      </a>
      <div className="sticky-nav-stack">
        <a href="/" className="sticky-nav-button">
          Blog
        </a>
        <a href="/" className="sticky-nav-button">
          CV
        </a>
      </div>
    </div>
  )
}

export default Header
