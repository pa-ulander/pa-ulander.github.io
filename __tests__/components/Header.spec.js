import React from 'react'
import { shallow } from 'enzyme'
import Header from './../../src/components/Header'

describe('Header component', () => {
  it('should render correctly', () => {
    const component = shallow(<Header />)
    expect(component).toMatchSnapshot()
  })
})
