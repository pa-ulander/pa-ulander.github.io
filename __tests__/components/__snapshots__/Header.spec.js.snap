// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render correctly 1`] = `
<div
  className="sticky-nav"
>
  <a
    className="sticky-nav-button"
    href="/"
  >
    Home
  </a>
  <div
    className="sticky-nav-stack"
  >
    <a
      className="sticky-nav-button"
      href="/"
    >
      Blog
    </a>
    <a
      className="sticky-nav-button"
      href="/"
    >
      Contact
    </a>
    <a
      className="sticky-nav-button"
      href="/"
    >
      About Us
    </a>
  </div>
</div>
`;
